tool/aligner_utils.py

```diff
@@ -189,8 +189,8 @@ def bowtie2_align_reads(
         ] + reads)
 
         cmd_view = ' '.join([
```

Comment:
should this still be cmd_view or cmd_sort ?

Reply:
These have now been renamed